components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/main/resources/org.wso2.carbon.identity.configuration.management.yaml

```diff
@@ -592,29 +592,29 @@ paths:
         - Attribute
     delete:
       description: |
-        This API is used to revoke a attribute in the tenant domain given by the user.
+        This API is used to revoke a attribute in the tenant domain for a given resource. 
       x-wso2-request: |
         curl -k -X DELETE "https://localhost:9443/api/identity/config-mgt/v1.0/resource/{resource-type}/{resource-name}/{attribute-key} -H "accept: application/json" -H "Authorization: Basic <Base64Encoded[username:password]>"
       x-wso2-curl: |
         curl -k -X DELETE 'https://localhost:9443/api/identity/config-mgt/v1.0/resource/e-mail/smtp/from' -H "accept: application/json" -H 'Authorization: Basic YWRtaW46YWRtaW4='
       x-wso2-response: |
         HTTP/1.1 200 Ok
       summary: |
-        Revoke the attribute
+        Revoke attribute
       parameters:
         - name: resource-name
           in: path
-          description:  This represents the name of the attribute to be retrieved.
+          description:  This represents the name of the attribute that is to be revoked.
           required: true
           type: 'string'
         - name: resource-type
           in: path
-          description:  This represents the type of the attribute to be deleted and can either be the name or id.
+          description:  This represents the type of attribute that is to be revoked and can either be the resource-type name or ID.
           required: true
           type: 'string'
         - name: attribute-key
           in: path
-          description:  This represents an attribute key of the attribute to be deleted.
+          description:  This represents a unique identifier of the attribute that is to be revokeda.
```

Comment:
Since the unique identifier would be the id value for this attribute, shall we change this to something like 'key of the attribute that is to be retrieved' ?

Reply:
Fixed with [c2af0a2](https://github.com/wso2/carbon-identity-framework/commit/c2af0a29f9313baf5105a4898a9e26baceae00fa)